js/test/transactions.spec.js

```diff
@@ -0,0 +1,169 @@
+        "f90343f9033fb9026c696d706f727420466c6f775374616b696e67436f6c6c656374696f6e2066726f6d203078386430653837623635313539616536330a0a2f2f2f204368616e67657320746865206e6574776f726b696e67206164647265737320666f722074686520737065636966696564206e6f64650a0a7472616e73616374696f6e286e6f646549443a20537472696e672c206e6577416464726573733a20537472696e6729207b0a202020200a202020206c6574207374616b696e67436f6c6c656374696f6e5265663a2026466c6f775374616b696e67436f6c6c656374696f6e2e5374616b696e67436f6c6c656374696f6e0a0a2020202070726570617265286163636f756e743a20417574684163636f756e7429207b0a202020202020202073656c662e7374616b696e67436f6c6c656374696f6e526566203d206163636f756e742e626f72726f773c26466c6f775374616b696e67436f6c6c656374696f6e2e5374616b696e67436f6c6c656374696f6e3e2866726f6d3a20466c6f775374616b696e67436f6c6c656374696f6e2e5374616b696e67436f6c6c656374696f6e53746f7261676550617468290a2020202020202020202020203f3f2070616e69632822436f756c64206e6f7420626f72726f772072656620746f205374616b696e67436f6c6c656374696f6e22290a202020207d0a0a2020202065786563757465207b0a202020202020202073656c662e7374616b696e67436f6c6c656374696f6e5265662e7570646174654e6574776f726b696e6741646472657373286e6f646549443a206e6f646549442c206e6577416464726573733a206e657741646472657373290a202020207d0a7d0af88eb85c7b2274797065223a22537472696e67222c2276616c7565223a2238383534393333356531646237623562343663326164353864646237306237613435653737306363356665373739363530626132366631306536626165356536227daf7b2274797065223a22537472696e67222c2276616c7565223a22666c6f772d6e6f64652e746573743a33353639227da0f0e4c2f76c58916ec258f246851bea091d14d4247a2fc3e18694461b1816e13b2a88f19c161bc24cf4b4040a88f19c161bc24cf4b4c988f19c161bc24cf4b4c0"
+    ]);
+
+    transactions
+        .filter((tx) => to_test.has(tx.encodedTransactionEnvelopeHex))
+        .forEach((tx) => {        
+            test(`sign transaction (${tx.title}) - ${ECDSA_P256.name} / ${SHA3_256.name}`, async () => {
+                const txExpectedPageCount = getTransactionPageCount(tx.envelopeMessage);
+
+                await transactionTest(
+                    tx.encodedTransactionEnvelopeHex, 
+                    txExpectedPageCount, 
+                    ECDSA_P256, 
+                    SHA3_256,
+                );
+            });
+        }, 100000);
+});
+
+describe("Show address", () => {
+    test("Test", async () => {
+        const transport = await TransportNodeHid.create();
+        const app = new FlowApp(transport);
+        const res = await app.showAddressAndPubKey("m/44\'/539\'/512\'/0/0")
+        console.log(res)
```

Comment:
shouldn't we assert something?

Reply:
Not necessarily, Zemu test do that. But it does no harm to put an assert there, so I'll do.

Reply:
These tests should work regardless of the mnemonics (to keep things as simple as possible). The aim is for you to see that happens on the ledger. The outputs are thoroughly tested in Zemu_ ests, so I rather favor simplicity here.